### ISSUE TITLE
Doctests don't simulate REPL :multiline

### ISSUE THREAD

**Issue by TotalVerb** [JuliaDocs/Documenter.jl#83]
On current v0.5 master, the Julia REPL displays output by `show`ing to an `IOContext` with `:multiline=true`. Documenter doesn't currently seem to simulate this:

Julia REPL:
```
julia> [1, 2, 3]
3-element Array{Int64,1}:
 1
 2
 3
```

But according to Documenter:
```
julia> [1, 2, 3]
[1,2,3]
```

I believe this only affects v0.5, as display worked very differently in v0.4.

**Comment by MichaelHatherly** [comment]
Thanks for finding this one, I was meaning to update the pacakge once the `:multiline` changes got merged. I'll update things later today.

**Comment by MichaelHatherly** [comment]
Fixed by #84.
